src/metabase/events/audit_log.clj

```diff
@@ -63,10 +63,11 @@
   [topic dashboard]
   (audit-log/record-event! topic dashboard))
 
-(derive ::table-read-event ::event)
-(derive :event/table-read ::table-read-event)
```

Comment:
renamed `::table-read-event` -> `::table-event` since that matches the pattern in this ns better